src/Paysera/WalletApi/Mapper.php

```diff
@@ -2036,12 +2036,10 @@ public function decodeClient(array $data)
         }
         $client->setHosts($hosts);
 
-        if (!empty($data['wallets_permissions'])) {
-            $walletsPermissions = [];
-            foreach ($data['wallets_permissions'] as $walletPermission) {
-                $walletsPermissions[] = $this->decodeClientWalletPermissions($walletPermission);
+        if (!empty($data['permissions_to_wallets'])) {
```

Comment:
https://github.com/paysera/php-style-guide#checking-things-explicitly

we use count($array) > 0 to check if array is not empty and not !empty($array), as we do not want to check whether $array is 0, false, '' or even not defined at all (in which case IDE would possibly hide some warnings that could help noticing possible bugs).

Reply:
In my case it can be undefined, so I need use if (isset($data['permissions_to_wallets']) && count($data['permissions_to_wallets']) > 0) ?

Reply:
Done